ArduPlane/commands_logic.cpp

```diff
@@ -261,7 +266,13 @@ bool Plane::verify_command(const AP_Mission::Mission_Command& cmd)        // Ret
         return quadplane.verify_vtol_takeoff(cmd);
 
     case MAV_CMD_NAV_VTOL_LAND:
-        return quadplane.verify_vtol_land();
+        if (quadplane.options & QuadPlane::OPTION_MISSION_LAND_FW_APPROACH && verify_landing_vtol_approach(cmd)) {
```

Comment:
I'd like verify_landing_vtol_approach() to be renamed to landing_vtol_approach_pending(), as other verify_*() commands return true when done, whereas this returns false when done

Reply:
AHHHHHH! No. That's a huge bug that crept in somewhere by accident. It absolutely expected to return true except in the complete case.... In fact it was just getting lucky that `control_auto()` wasn't being called.

Reply:
More parens? Always!